draft-ietf-quic-qpack.md

```diff
@@ -179,11 +191,11 @@ blocking if the encoder has not received an acknowledgement indicating the entry
 is available at the decoder.
 
 An encoder MAY insert any entry in the dynamic table it chooses; it is not
-limited to header fields it is compressing.
+limited to field lines it is compressing.
 
-QPACK preserves the ordering of header fields within each header list.  An
-encoder MUST emit header field representations in the order they appear in the
-input header list.
+QPACK preserves the ordering of field lines within each field section.  An
+encoder MUST emit field representations in the order they appear in the input
+field list.
```

Comment:
Maybe "field section" instead of "field list", because "field list" is not defined in SEMANTICS.